hack/verify-internal-modules.sh

```diff
@@ -26,7 +26,7 @@ kube::util::ensure_clean_working_dir
 kube::golang::setup_env
 
 _tmpdir="$(kube::realpath "$(mktemp -d -t verify-internal-modules.XXXXXX)")"
-#kube::util::trap_add "rm -rf ${_tmpdir}" EXIT
+kube::util::trap_add "rm -rf ${_tmpdir:?}" EXIT
```

Comment:
@thockin lost your LGTM pulling this one out

Also adding `:?` because even though we just set _tmpdir we should encourage the habit of `rm -rf ${foo:?}` to prevent accidents like `rm -rf "${foo:}/"` where $foo is unset.